packages/cf.js/considerations-for-0-0-2.md

```diff
@@ -0,0 +1,13 @@
+# missing features of v0.0.2
+
+- need a way to provide a `timeout` during the install flow
+    - argument to `proposeInstall`?
+    - attribute of the manifest.json
+- currently, the user's address is never being set
```

Comment:
Why do we need this? Per-app user addresses should be generated deterministically by the node.

Reply:
Currently, we provide the `fromAddress` in all action messages to the node, and I suspect we'll still want to, right? Or will the next generation of the node not need this information because it's able to deterministically generate that address?

Reply:
I would definitely think the latter. The node should be able to generate or recall `fromAddress` based on just the appId.

Reply:
That would be cool. It might be worth hashing out with @cf19drofxots what our node requests should look like.

Reply:
What is the context here? Where is the user's address not being set?

And yes, in the next version of the Node, addresses are deterministically generated per-app via ephemeral signing keys.

Reply:
The user's address is never being set in the client interface, though it seems like that will no longer be necessary. We were using it previously to provide the `fromAddress` in our action requests, but it sounds like the node will won't need that information any longer.